### ISSUE TITLE
Cannot import package on Mac

### ISSUE THREAD

**Issue by tarensanders** [scholarly-python-package/scholarly#112]
Possibly related to #111. 

Installing from GitHub (not PyPi) and importing on a Mac causes an AttributeError.

```
from scholarly import scholarly


---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-97c1d4197e75> in <module>
      4 from wos import WosClient
      5 import wos.utils
----> 6 from scholarly import scholarly
      7 
      8 import pandas as pd

/opt/anaconda3/lib/python3.7/site-packages/scholarly/__init__.py in <module>
      1 from ._scholarly import _Scholarly
      2 
----> 3 scholarly = _Scholarly()

/opt/anaconda3/lib/python3.7/site-packages/scholarly/_scholarly.py in __init__(self)
     11     """docstring for scholarly"""
     12     def __init__(self):
---> 13         self.nav = Navigator()
     14 
     15     def use_proxy(self, http: str, https: str):

/opt/anaconda3/lib/python3.7/site-packages/scholarly/_navigator.py in __call__(cls, *args, **kwargs)
     37         if cls not in cls._instances:
     38             cls._instances[cls] = super(Singleton, cls).__call__(*args,
---> 39                                                                  **kwargs)
     40         return cls._instances[cls]
     41 

/opt/anaconda3/lib/python3.7/site-packages/scholarly/_navigator.py in __init__(self)
     50         self._tor = False
     51         self._proxy = False
---> 52         self._setup_tor()
     53         self._TIMEOUT = 10
     54 

/opt/anaconda3/lib/python3.7/site-packages/scholarly/_navigator.py in _setup_tor(self)
     65             self._TOR_CONTROL = 9151
     66 
---> 67         self.proxies = {'http': self._TOR_SOCK,
     68                         'https': self._TOR_SOCK}
     69 

AttributeError: 'Navigator' object has no attribute '_TOR_SOCK'
```

It looks like the values _TOR_SOCK are only created on Linux and Windows machines. I actually don't even want to use Tor, but the _setup_tor() is called anyway.

To be slightly more helpful: ln 55 of _navigator.py
```
    def _setup_tor(self):
        """Initialized ToR Proxy"""

        # Tor uses the 9050 port as the default socks port
        # on windows 9150 for socks and 9151 for control
        if sys.platform.startswith("linux"):
            self._TOR_SOCK = "socks5://127.0.0.1:9050"
            self._TOR_CONTROL = 9051
        elif sys.platform.startswith("win"):
            self._TOR_SOCK = "socks5://127.0.0.1:9150"
            self._TOR_CONTROL = 9151

        self.proxies = {'http': self._TOR_SOCK,
                        'https': self._TOR_SOCK}

        self._tor = self._proxy_works()
```

On Mac, `sys.platform` returns "darwin". Hence _TOR_SOCK is not set. I would offer a fix, but I don't know anything about Tor.

**Comment by ipeirotis** [comment]
I just merged #111 that should fix the issue. Could you please try again?

**Comment by tarensanders** [comment]
Much appreciated. It imports correctly now.
